test/e2e/manifest.yaml

```diff
@@ -0,0 +1,29 @@
+# Test manifest for https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external
+ShortName: manila
+StorageClass:
+  FromExistingClassName: standard-csi
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: manila.csi.openstack.org
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 16Ti
+  SupportedFsType:
+    xfs: {}
+    ext4: {}
```

Comment:
I think this should be empty for manila, otherwise the tests will ask for XFS and ext4 volumes, which manila should not provide.

Reply:
ack, I'll remove `SupportedFsType`.